src/main/protobuf/soc/board.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto2";
+
+package protos.soc;
+
+message Hex {
+    enum Resource {
+        BRICK = 0;
+        ORE = 1;
+        SHEEP = 2;
+        WHEAT = 3;
+        WOOD = 4;
+        DESERT = 5;
+    }
+    required string id = 1;
```

Comment:
Oh wow, didn't know there was a `required` key word. Need to start updating my PRs to include this!

Reply:
Im using proto2 instead of proto3 so I can use requred and optional. I think in proto3 everything is optional